### DISCUSSION TITLE
QTable, Allow not all rows to be selectable

### DISCUSSION THREAD

**Discussion by iliubinskii** [quasarframework/quasar#12591] [Ideas / Proposals]
I suggest to add "selectable" prop that will accept a function to determine whether row is selectable.

**Comment by MilosPaunovic** [comment]
You can already achieve that easily with selection cell slots:
[https://quasar.dev/vue-components/table#example--selection-cell-slots](https://quasar.dev/vue-components/table#example--selection-cell-slots)

[https://codepen.io/milospaunovic/pen/gOXdopE?editors=1010](https://codepen.io/milospaunovic/pen/gOXdopE?editors=1010)
